src/components/common/ColonyCreationWizard/StepTokenChoice.tsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { useMediaQuery } from 'react-responsive';
+
+import { WizardStepProps } from '~shared/Wizard';
+import Heading from '~shared/Heading';
+import ExternalLink from '~shared/ExternalLink';
+import DecisionHub from '~shared/DecisionHub';
+import { Form } from '~shared/Fields';
+import { multiLineTextEllipsis } from '~utils/strings';
+import { SELECT_NATIVE_TOKEN_INFO } from '~constants';
+
+import { FormValues, Step2 } from './ColonyCreationWizard';
+
+import { query700 as query } from '~styles/queries';
+import styles from './StepTokenChoice.css';
+
+const LEARN_MORE_URL = SELECT_NATIVE_TOKEN_INFO;
```

Comment:
Is this needed ? can't you just use `SELECT_NATIVE_TOKEN_INFO` directly ?

Or if you really care about naming, do something like:
```ts
import { SELECT_NATIVE_TOKEN_INFO as LEARN_MORE_URL } from '~constants';
```